.prettierignore

```diff
@@ -1,5 +1,7 @@
 /dist
 /modules/schematics/src/*/files/*
+/modules/schematics-core/templates/*
```

Comment:
Change `templates` to `files` so its consistent with schematics. If we want to change `files` to `templates` across the board, we'll do that in a separate PR.